### ISSUE TITLE
Failed to connect to remote target

### ISSUE THREAD

**Issue by ksoiza** [OneLiberty/moonlight-chrome-tizen#63]
Hi,

I think I've followed just about every step to install moonlight on my Samsung S90C but still seem to be running into this issue when trying to execute the final step and install moonlight on the TV.
![Screenshot 2024-09-15 192249](https://github.com/user-attachments/assets/51c3eb6b-5916-4892-9b8f-aa28e1651d4e)

I think I haven't missed any step but stumped here.

Edit- I've also attempted the USB install but it says "app cannot be installed because the validity period has expired."

**Comment by MrPhaze62** [comment]
Make sure dev mode is ENABLED and has your PC ip address on your tv and then restart the tv by pressing the power button for 2 second after doing so.

And then try again with the correct tv ip address in the cmd.

**Comment by ksoiza** [comment]
> Make sure dev mode is ENABLED and has your PC ip address on your tv and then restart the tv by pressing the power button for 2 second after doing so.
> 
> And then try again with the correct tv ip address in the cmd.

Yup, dev mode is indeed enabled and checked the PC IP address multiple times to ensure it wasn't that. 

My only guess after almost 3 hours on this yesterday is it might have to do with ports? My ISP doesn't provide me with access to my router and I can't manually open or check ports myself.

**Comment by OneLiberty** [comment]
Well this could in fact be the reason, have you tried bridging TV and PC with a phone connection without using the router ?

**Comment by ksoiza** [comment]
> Well this could in fact be the reason, have you tried bridging TV and PC with a phone connection without using the router ?

I made the rookie mistake of getting a PC with no Wifi so need to see if I can borrow a friend's Wifi antenna and try that. Thanks for the suggestion though, if I manage to get my hands on one I'll update on the results! :)

**Comment by OneLiberty** [comment]
No problem ! 
Btw some phones allow you to share network with a cable (usb hotspot ?) and WiFi, so you could still try connecting the phone to the PC with a cable and the TV via WiFi

**Comment by eduardomozart** [comment]
Hello, I'm facing this same problem on a UN43T5300AGXZD TV and before (Aug 27 2024) I was able to. It doesn't seems to be a TV firmware update as it's the same than before (2503), only I upgraded my macOS to Sequoia and installed Mcafee that has a built-in firewall - disabled it but the issue persists. I did a Wireshark capture and running the ``sdb connect`` command doesn't seem to send any packet to the TV.
Also, I'm not able to use ``ping`` command inside the Docker container, but my PC is able to ping the TV. If the ``ping`` and ``traceroute`` commands be available inside Docker container it would be nice.
Edit: I figure it out what was going on. mac Sequoia requires to enable "Local network" on Privacy applet to allows Docker to connect to LAN devices:
<img width="794" alt="image" src="https://github.com/user-attachments/assets/ac1032e3-6dcc-44f4-a5cc-a563c33ef3fe">

**Comment by martinderm** [comment]
Same problem on Windows 11. Any tipps?

`docker run -it --rm ghcr.io/oneliberty/moonlight-chrome-tizen:samsung_wasm
moonlight@5dde42360621:~$ sdb connect 192.168.1.193
* Server is not running. Start it now on port 26099 *
* Server has started successfully *
connecting to 192.168.1.193:26101 ...
failed to connect to 192.168.1.193:26101
moonlight@5dde42360621:~$`

**Comment by sancel22** [comment]
I fixed mine by connecting to the TV's assigned IP address instead of the PC host's IP address.

**Comment by ksoiza** [comment]
> No problem ! Btw some phones allow you to share network with a cable (usb hotspot ?) and WiFi, so you could still try connecting the phone to the PC with a cable and the TV via WiFi

Wasn't able to look back into it until I got back from holiday a couple days back.

The issue was indeed that my ports weren't fully open. Got my ISP to open the ports for one of my connections and connected to that one, worked like a charm. Thanks for the help!
